### PR TITLE
HOMEWORK-0

### PR DESCRIPTION
**Pull request by erinleeny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# nha-erinlee
\ No newline at end of file
+# nha-erinlee
+for the new semester(APIs) for nanohacker academy.
```
Comment (rickyc) [inline README.md:2]: You didn't mention me with the @ to let me know this was done 😱 . Next step would be to fix the merge conflict.

Comment (erinleeny) [inline README.md:2]: Oh. hehe. Oopsies. I don’t have atom yet.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
+<<<<<<< HEAD
+# nha-erinlee
+for the new semester(APIs) for nanohacker academy.
+=======
 # nha-erinlee 
 this is for the new repostiory for the new semester.
+>>>>>>> master
```
Comment (rickyc) [inline README.md]: 😅 . Okay! So you see the `<<<<<<`, `=====` and `>>>>>>`? This means there is a conflict with the file. Lines 2, 3 are code from the `homework-0` branch. Notice how it's above the `=====`. HEAD means your current commit. Lines 5, 6 are on the `master` branch; you'll notice that line 7 is telling you that as well. 

What you want to do now is look at which lines you want to keep and delete the rest of the code that you don't want. Let me know when that's done.

**Comment by rickyc** [comment]
👍  LGTM! Okay, you can click the green button and merge it in, but make sure you change the green button to say `Confirm squash and merge`. You can bring up this text by clicking on the arrow next to the green merge button. This will basically combine all your commits to a single commit when it pushes to master.

https://github.com/blog/2141-squash-your-commits

**Comment by rickyc** [comment]
Oh yeah, I forgot. LGTM means `looks good to me`. It's something people will often write on pull requests. You just completed the pull request flow! Congrats! This is the workflow that engineers use when they commit code.

**Comment by erinleeny** [comment]
Mine doesnt have an arrow though.

**Comment by rickyc** [comment]
Ahh, just hit the green merge button :). You can delete the branch afterwards. Make sure you switch back to the master branch on your GitHub desktop app and press the sync to pull the most up to date information.

**Comment by erinleeny** [comment]
oh wait- i see the arrow. do i still have to do the second instruction?

**Pull request merged**